protocol/avdl/keybase1/kbfsmount.avdl

```diff
@@ -0,0 +1,6 @@
+@namespace("keybase.1")
+protocol kbfsMount {
+    string GetCurrentDriveLetter();
```

Comment:
Maybe update these all to be `GetCurrentMountDir` / `GetAllAvailableMountDirs` and `SetCurrentMountDir`?